### ISSUE TITLE
Verification number not displayed correctly, when using a larger font

### ISSUE THREAD

**Issue by mueller-ma** [signalapp/Signal-Android#8083]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Verification number not displayed correctly, when using a larger font.

See #7251

### Steps to reproduce
- Android Settings => Accessibility
- Set font size and display size to large and enable high contrast text.
- Go to signal and display verification number.

**Actual result:** The most left and right numbers are cut, QR code is ok.
**Expected result:** The number is fully readable.

### Screenshots
![](https://user-images.githubusercontent.com/22525368/33673052-0e445646-daac-11e7-8b8d-e75bd0c77e79.png)

### Device info

**Device:** Oneplus X
**Android version:** 7.1.2, LineageOS 14.1
**Signal version:** 4.23.4

**Device:** Moto G4
**Android version:** 7.1.2, LineageOS 14.1
**Signal version:** 4.23.4

**Comment by mueller-ma** [comment]
This is still an issue.
